packages/http/src/helpers/fetch.requestHandler.helper.ts

```diff
@@ -0,0 +1,9 @@
+import { merge } from 'lodash';
+import { NormalizedHttpOptions } from '../httpClient.types';
+
+export const mergeRequestOptions = (
```

Comment:
It's too specific, since it only applies to `fetch.requestHandler`. I'd put it directly in `fetch.requestHandler`, because here, in `helpers` we only keep utilities that apply to all components or components in package root.

Reply:
Alright, makes sense - will move this function to proposed place.